tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -4,20 +4,23 @@ kind: Task
 metadata:
   name: rh-sign-image
   labels:
-    app.kubernetes.io/version: "3.4.1"
+    app.kubernetes.io/version: "3.5.0"
```

Comment:
New mandatory parameter is breaking change - 4.0.0 please

Reply:
fixed

Reply:
added comment in changelog too